nservicebus/upgrades/acs-host-7to8.md

```diff
@@ -12,7 +12,19 @@ upgradeGuideCoreVersions:
  - 7
 ---
 
-The NServiceBus Azure Host will be deprecated as of Version 9 and users are recommended to switch to self hosting for new endpoints. Upgrading existing endpoints is still supported for Version 8
+The NServiceBus Azure Host will be deprecated as of Version 9 and users are recommended to switch to self hosting for new endpoints. Upgrading existing endpoints is still supported for Version 8.
+
+The following are the main reasons why we are deprecating the NServiceBus Host:
+
+- Ease of configuration: Endpoint configuration and related APIs have been significantly improved over the last few years, therefore self-hosting is much simpler to set up than it used to be when Host was first introduced.
+
+- Ease and transparency of deployment: The Host under the hoods used the `sc.exe` tool, but required performing additional manual steps to ensure reliability. Those steps were described in the documentation but were sometimes missed, which could cause problems at runtime. With the new recommended [installation procedure](/nservicebus/hosting/windows-service.md#installation) users are guided through the process and less likely to omit crucial configuration steps.
```

Comment:
>The Host under the hoods used the `sc.exe` tool

Not correct, we use TopShelf that in turn calls into https://msdn.microsoft.com/en-us/library/system.serviceprocess.serviceinstaller.install(v=vs.110).aspx

see https://github.com/Particular/Topshelf/blob/master/src/Topshelf/Hosts/AbstractInstallerHost.cs

Perhaps we can just say that instead of abstracting Service installation we now "force" them to be more explicit about how to install their service?

Reply:
eh... ok :)